components/hubspot/hubspot.app.js

```diff
@@ -3,226 +3,221 @@ const axios = require("axios");
 module.exports = {
   type: "app",
   app: "hubspot",
+  propDefinitions: {
+    lists: {
+      type: "string[]",
+      label: "Lists",
+      description: "Select the lists to watch for new contacts.",
+      async options(prevContext) {
+        const { offset } = prevContext;
+        const params = {
+          count: 250,
+          offset: offset || 0,
+        };
+        const results = await this.getLists(params);
+        const options = results.map((result) => {
+          const { name: label, listId } = result;
+          return {
+            label,
+            value: JSON.stringify({ label, value: listId }),
```

Comment:
I included `label` so that I could use it in generateMeta:
```
    generateMeta(contact, list) {
      const { vid, properties } = contact;
      const { value, label } = list;
      return {
        id: `${vid}${value}`,
        summary: `${properties.firstname.value} ${properties.lastname.value} added to ${label}`,
        ts: Date.now(),
      };
    },
```

Reply:
Yeah, that makes sense given the fact that otherwise we'll have to do a separate call to retrieve that.

Just be aware that any type/data restrictions applied to that prop (e.g. min/max, length, etc) can no longer be applied, as we're storing a serialized version of an object. If a user overrides the value with a string like `foo bar`, then the call to `JSON.parse` will throw an exception.